package.json

```diff
@@ -2,10 +2,11 @@
   "name": "meriyah",
   "version": "4.1.1",
   "description": "A 100% compliant, self-hosted javascript parser with high focus on both performance and stability",
-  "main": "dist/meriyah.umd.js",
+  "main": "dist/meriyah.umd.cjs",
   "module": "dist/meriyah.esm.js",
   "jsnext:main": "dist/meriyah.esm.js",
   "browser": "dist/meriyah.umd.js",
+  "unpkg": "dist/meriyah.umd.js",
   "type": "module",
```

Comment:
should remove `"type": "module"` - as it also has cjs?

Reply:
No, `"type": "module"` just means that `.js` files will be loaded as ES modules. It doesn't prevent you from also shipping `cjs`. In fact the `.cjs` ending is more useful when you have `"type": "module"` because then you're specifying that specific file should be loaded as CommonJS and not as the ES default specified by `"type": "module"`

Reply:
I see.  just feel a little counter-intuitive for both shipping cjs and esm.

consider renaming `.esm.js` => `.mjs` , just like `.cjs`?

Reply:
It's pretty common to ship both multiple versions so that the appropriate one is available for each consumer. I think it makes sense how it is

Reply:
to clarify: it is  common to ship both multiple versions, but a little counter-intuitive to have `"type": "module"` in this kind of packages. 😄

Reply:
well. I think I was sold!